README.md

```diff
@@ -86,3 +86,18 @@ export VSCODE_EXTENSIONS_DIR="$HOME/.vscode-insiders/extensions"
 By default, VS Code changes on the remote server won't be synced back
 when the connection closes. To synchronize back to local when the connection ends,
 pass the `-b` flag.
+
+
+### Auto open in browser trouble shooting
+
+sshcode detects if you're under GUI mode via checking environment variable `DISPLAY`.
```

Comment:
Maybe something like 

```
sshcode determines whether to open an IDE window based on the existence of the `DISPLAY` environment variable.  If the `DISPLAY` variable is unset or has an empty value then it will not attempt to open a window. 

For some environments where X-Window is not installed (e.g.: MacOS), you will need to export DISPLAY to some value.

```bash
export DISPLAY=":0"
```
```